docs/examples/ros2_compatibility.rst

```diff
@@ -21,6 +21,13 @@ ROS 2 Shapes Demo main window changes to reflect this by showing a different tit
    :alt: ROS 2 Shapes Demo Main Window
    :align: center
 
+.. note::
+   When eProsima Shapes Demo is compiled with ROS 2, the
+   `Shapes Demo TypeSupport <https://github.com/eProsima/ShapesDemo-TypeSupport>`_ (see
+   :ref:`linux_ros2_compilation` compilation section) and the ROS 2 Shapes Demo executable is launched in a context with a valid ROS 2 installation sourced,
+   the default value of the "Use ROS 2 Topics" checkbox will be set to true.
+   Otherwise, if a valid ROS 2 installation could not be found, the checkbox will remain disabled.
```

Comment:
```suggestion
   Otherwise, the checkbox will remain disabled.
```
I don't think it's a matter of whether a ROS 2 installation can be found or not, but rather if it has been sourced or not.